### PR TITLE
feat: add internationalization i18n

### PR DESCRIPTION
**Pull request by Kovalenko77**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,8 +13,12 @@
     "@types/node": "^20.11.17",
     "@types/react": "^18.2.55",
     "@types/react-dom": "^18.2.19",
+    "i18next": "^23.10.1",
+    "i18next-browser-languagedetector": "^7.2.0",
```
Comment (romanalesenkov1) [inline package.json]: do you use this library?

File: src/Form.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
-
+import { useTranslation } from 'react-i18next';
```
Comment (romanalesenkov1) [inline src/Form.tsx:5]: add a space line here

File: src/Posts.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
```
Comment (romanalesenkov1) [inline src/Posts.tsx]: what is this file for?

File: src/i18n.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import i18n from 'i18next';
```
Comment (romanalesenkov1) [inline src/i18n.js]: refactor this file to TypeScript

File: src/Comments.tsx [hunk 5]
```diff
@@ -18,6 +19,8 @@ const Comments = () => {
   const [isDataLoading, setIsDataLoading] = useState(false);
   const [hasError, setHasError] = useState(false);
 
+  const { t } = useTranslation();
```
Comment (romanalesenkov1) [inline src/Comments.tsx]: it is better to write it at the top of the component

File: package.json (hunk 1 above)
Comment (Kovalenko77) [reply package.json]: No.Deleted.Thanks.

File: src/Posts.tsx (hunk 3 above)
Comment (Kovalenko77) [reply src/Posts.tsx]: I will move it to another project. Thanks.

File: src/Comments.tsx (hunk 5 above)
Comment (Kovalenko77) [reply src/Comments.tsx]: Done.Thanks.

File: src/Form.tsx (hunk 2 above)
Comment (Kovalenko77) [reply src/Form.tsx:5]: Done.Thanks.

File: src/i18n.js (hunk 4 above)
Comment (Kovalenko77) [reply src/i18n.js]: Done.Thanks.

File: package-lock.json [hunk 6]
```diff
@@ -18,8 +18,12 @@
         "@types/node": "^20.11.17",
         "@types/react": "^18.2.55",
         "@types/react-dom": "^18.2.19",
+        "i18next": "^23.10.1",
+        "i18next-browser-languagedetector": "^7.2.0",
```
Comment (romanalesenkov1) [inline package-lock.json]: do you need this package?

File: src/Home.tsx [hunk 7]
```diff
@@ -21,15 +21,17 @@ const Home = ({
...
-    <Form
-      timeSpentEnglish={timeSpentEnglish}
-      timeSpentIT={timeSpentIT}
-      date={date}
-      setDate={setDate}
-      setTimeSpentIT={setTimeSpentIT}
-      setTimeSpentEnglish={setTimeSpentEnglish}
-      submit={submit}
-    />
+    <>
```
Comment (romanalesenkov1) [inline src/Home.tsx]: why do you need this change?

File: src/i18n.tsx [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import i18n from 'i18next';
```
Comment (romanalesenkov1) [inline src/i18n.tsx]: .tsx is only for files, which contain JSX syntax + Typescript.
If you use only typescript, then you should you .ts extension.

Comment (Kovalenko77) [reply src/i18n.tsx]: Done.Thanks.

File: src/Home.tsx (hunk 7 above)
Comment (Kovalenko77) [reply src/Home.tsx]: Deleted.

File: package-lock.json (hunk 6 above)
Comment (Kovalenko77) [reply package-lock.json]: No.Thanks.

**Pull request merged**